test/pylib/scylla_cluster.py

```diff
@@ -1597,6 +1611,7 @@ def add_put(route: str, handler: Callable):
         add_get('/cluster/replicas', self._cluster_replicas)
         add_get('/cluster/running-servers', self._cluster_running_servers)
         add_get('/cluster/all-servers', self._cluster_all_servers)
+        add_get('/cluster/starting-servers-ids', self._cluster_starting_servers_ids)
```

Comment:
I would better use `starting-servers` to have consistency across other endpoints, 
even if you need only ids and named it accordingly, and return only ids in manager client.

Reply:
On second look, I think that ServerInfo is benign enough - containing the id, ips, datacenter and rack - that it's ok to return it also for starting servers. I'll do it.